scripts/install-deps.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+echo "deb http://packages.laboratoriopublico.org/publiclab/ stretch main" > /etc/apt/sources.list.d/publiclab.list
+
+# Obtain key
+mkdir -p ~/.gnupg
+echo "disable-ipv6" >> ~/.gnupg/dirmngr.conf
+apt-key adv --keyserver ipv4.pool.sks-keyservers.net --recv-keys BF26EE05EA6A68F0
+add-apt-repository -y ppa:ubuntugis/ppa
+
+# Install dependencies
+apt-get update -qq && apt-get install -y \
+                        gdal-bin \
+                        python3-gdal \
+                        python-gdal \
+                        libgdal-dev \
+                        g++ \
+                        curl \
+                        procps \
+                        git \
+                        imagemagick \
+                        zip
+
+sed -i 's/<policy domain="delegate" rights="none" pattern="HTTPS" \/>//g' /etc/ImageMagick-6/policy.xml
```

Comment:
This is a security measure that should not be relevant in .travis.yml as it doesn't process outside requests.
The goal of this PR is to improve test times, yes?
If it does, looks good to me!

Reply:
Great, I didn't quite know what was going on here, thx for pointing that out. I'll make those adjustments